README.md

```diff
@@ -8,9 +8,7 @@
 [![GitHub pull requests](https://img.shields.io/github/issues-pr-raw/influxdata/influxdb-client-csharp.svg)](https://github.com/influxdata/influxdb-client-csharp/pulls)
 [![Slack Status](https://img.shields.io/badge/slack-join_chat-white.svg?logo=slack&style=social)](https://www.influxdata.com/slack)
 
-This repository contains the reference C# client for the InfluxDB 2.x.
-
-#### Note: Use this client library with InfluxDB 2.x and InfluxDB 1.8+ ([see details](#influxdb-18-api-compatibility)). For connecting to InfluxDB 1.7 or earlier instances, use the [influxdb-csharp](https://github.com/influxdata/influxdb-csharp) client library.
+This repository contains the C# client library for use with InfluxDB 2.0 specifically with Flux. For InfluxDB 3.0 users, utilize the lightweight [v3 client library](https://github.com/InfluxCommunity/influxdb3-csharp). Most users of previous versions of InfluxDB will want to use InfluxQL and the [v1 client library](https://github.com/influxdata/influxdb-csharp).
```

Comment:
Thanks for the PR! 

Instead of "previous versions" could we say 1.0? I might suggest we avoid saying InfluxQL with 1.x since it is supported again in 3.x.

```suggestion
This repository contains the C# client library for use with InfluxDB 2.0 specifically with Flux. For InfluxDB 3.0 users, utilize the lightweight [v3 client library](https://github.com/InfluxCommunity/influxdb3-csharp). 1.x users  will want to use the [v1 client library](https://github.com/influxdata/influxdb-csharp).
```